taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+
+from django.urls import path
+from taxi.views import index

```

Comment:
No need for blank line at the start of the file. But add blank line between different groups of imports according to PEP8